pkg/kubectl/cmd/auth/reconcile.go

```diff
@@ -161,13 +166,13 @@ func (o *ReconcileOptions) RunReconcile() error {
 		}
 
 		switch t := info.Object.(type) {
-		case *rbac.Role:
+		case *rbacv1.Role:
```

Comment:
does this mean `kubectl auth reconcile` only works on rbac/v1 resources? that's notable

Reply:
> does this mean kubectl auth reconcile only works on rbac/v1 resources? that's notable

Yes.  It is consistent with the idea of kubectl moving to external versions and skewing only by a release.